drivers/timex.js

```diff
@@ -0,0 +1,44 @@
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat, flatMap } = require('../lib/utils.js');
+
+module.exports = async () => {
+  const tickers = await request(
+    'https://plasma-relay-backend.timex.io/public/tickers?period=H1',
```

Comment:
Can we change this to this URL: `https://plasma-relay-backend.timex.io/public/tickers24?useCache=true` because the H1 period is not a 24h time period